### PR TITLE
replaced tryocaml references with playground in doc

### PR DESCRIPTION
**Pull request by SaySayo**
I replaced the TryOcaml references in the "A First Hour With OCaml" Document with the official [OCaml Playground](https://ocaml.org/play) as mentioned in issue (#738)

### CODE REVIEW

File: data/tutorials/gs_01_a_first_hour_with_ocaml.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ You may follow along with this tutorial with just a basic OCaml installation,
 as described in [Up and Running](/docs/up-and-running).
 
 Alternatively, you may follow almost all of it by running OCaml in your browser
-using [TryOCaml](http://try.ocamlpro.com), with no installation required.
+using [Playground](https://ocaml.org/play), with no installation required.
```
Comment (sabine) [inline data/tutorials/gs_01_a_first_hour_with_ocaml.md]:
```suggestion
using [the OCaml Playground](https://ocaml.org/play), with no installation required.
```

File: data/tutorials/gs_01_a_first_hour_with_ocaml.md [hunk 2]
```diff
@@ -53,10 +53,10 @@ utop # 50 * 50;;
 - : int = 2500
 ```
 
-The in-browser [TryOCaml](http://try.ocamlpro.com) has a similar interface.
+The in-browser [Playground](https://ocaml.org/play) has a similar interface.
```
Comment (sabine) [inline data/tutorials/gs_01_a_first_hour_with_ocaml.md]:
```suggestion
The in-browser [OCaml Playground](https://ocaml.org/play) has a similar interface.
```

File: data/tutorials/gs_01_a_first_hour_with_ocaml.md [hunk 3]
```diff
@@ -53,10 +53,10 @@ utop # 50 * 50;;
 - : int = 2500
 ```
 
-The in-browser [TryOCaml](http://try.ocamlpro.com) has a similar interface.
+The in-browser [OCaml Playground](https://ocaml.org/play) has a similar interface.
 
 The examples in this tutorial can be typed in by hand, or copied into `ocaml`,
```
Comment (christinerose) [inline data/tutorials/gs_01_a_first_hour_with_ocaml.md]:
```suggestion
The examples in this tutorial can be typed in by hand or copied into `ocaml`,
```

**Review by christinerose — APPROVED** [review]
Looks great! Just one comma removal suggestion. :)

**Pull request merged**